sdbuild/scripts/create_bsp.sh

```diff
@@ -6,6 +6,13 @@ set -e
 board=$1
 template=$2
 
+# If $BSP is a URL, fetch it!
+if [[ "$BSP" == *"://"* ]] ; then
+       BSP_NEW=`basename "$BSP"`
```

Comment:
This will provide unreliable filename in case a the url doesn't contain the actual filename (e.g. if using a url shortener)
maybe it's worth considering using `$BSP_PROJECT.bsp` (see [here](https://github.com/Xilinx/PYNQ/blob/master/sdbuild/Makefile#L100) and two lines below that) especially if you use a dedicated folder like `BSP_ABS=$BSP_BUILD/../downloaded_bsp` as I am suggesting below.

Won't affect functionality, but definitely readability will be improved: better to have predictable places and names for things, will help looking for stuff in case needed/debugging.

Reply:
I've changed the filename to be set to downloaded-${BSP_PROJECT}.bsp.   I don't see the point in creating a separate directory for it given that the file is download into the throwaway per-board build directory already.